test/CleanCommand.js

```diff
@@ -10,59 +10,103 @@ import stub from "./_stub";
 import assertStubbedCalls from "./_assertStubbedCalls";
 
 describe("CleanCommand", () => {
-  let testDir;
 
-  beforeEach((done) => {
-    testDir = initFixture("CleanCommand/basic", done);
-  });
+  describe("basic tests", () => {
+    let testDir;
 
-  it("should rm -rf the node_modules", (done) => {
-    const cleanCommand = new CleanCommand([], {});
+    beforeEach((done) => {
+      testDir = initFixture("CleanCommand/basic", done);
+    });
 
-    assertStubbedCalls([
-      [PromptUtilities, "confirm", { valueCallback: true }, [
-        { args: ["Proceed?"], returns: true }
-      ]],
-    ]);
+    it("should rm -rf the node_modules", (done) => {
+      const cleanCommand = new CleanCommand([], {});
 
-    cleanCommand.runValidations();
-    cleanCommand.runPreparations();
+      assertStubbedCalls([
+        [PromptUtilities, "confirm", { valueCallback: true }, [
+          { args: ["Proceed?"], returns: true }
+        ]],
+      ]);
 
-    let curPkg = 1;
-    stub(FileSystemUtilities, "rimraf", (actualDir, callback) => {
-      const expectedDir = path.join(
-        testDir, "packages/package-" + curPkg, "node_modules"
-      );
+      cleanCommand.runValidations();
+      cleanCommand.runPreparations();
 
-      assert.equal(actualDir, expectedDir);
+      let curPkg = 1;
+      stub(FileSystemUtilities, "rimraf", (actualDir, callback) => {
+        const expectedDir = path.join(
+          testDir, "packages/package-" + curPkg, "node_modules"
```

Comment:
Added `path.join` down around line 128. I also realized I oopsidentally checked in a `describe.only`, so I removed that too :grin: